Dockerfile

```diff
@@ -1,8 +1,18 @@
-FROM node:lts-alpine
-
+FROM node:lts-alpine AS base
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
 WORKDIR /app
 
-RUN apk --no-cache add git
-RUN npm install --global pnpm
+RUN npm install -g npm@latest
```

Comment:
No, just preference as it was used to install pnpm.  
Which is using the official script now, not npm. Removed updating npm.

On that note:  
I pinned the version of pnpm as well to avoid future potential breaks as with npm 9 -> 10.
For now to the version specified in `package.json`.  

This is not a requirement. It will install the version from `package.json` which is then accessible in `/app` anyway.
For that I moved the install of pnpm into the dependency stage of `Dockerfile`.
Inside of `/app` `pnpm --version` now returns the version specified under `package.json`.